netbox_librenms_plugin/templates/netbox_librenms_plugin/htmx/_oob_interface_select.html

```
@@ -0,0 +1,32 @@
+oob_default_new_name.
+{% endcomment %}
+<div class="mb-2 text-start">
+  <label class="form-label small mb-1" for="oob-iface-{{ libre_device.device_id }}">OOB IP interface</label>
+  <select name="oob_interface_id" id="oob-iface-{{ libre_device.device_id }}" class="form-select form-select-sm"
+          onchange="var creating = this.value === '__new__'; document.getElementById('oob-new-iface-{{ libre_device.device_id }}').style.display = creating ? 'block' : 'none'; var nameInput = document.getElementById('oob-new-iface-name-{{ libre_device.device_id }}'); nameInput.disabled = !creating; nameInput.required = creating;">
+    {% for iface in oob_interfaces %}
+      <option value="{{ iface.pk }}" {% if iface.pk == oob_suggested_interface_id %}selected{% endif %}>{{ iface.name }}</option>
+    {% endfor %}
+    <option value="__new__" {% if not oob_suggested_interface_id %}selected{% endif %}>+ Create new interface…</option>
+  </select>
+  <div id="oob-new-iface-{{ libre_device.device_id }}" class="mt-1"{% if oob_suggested_interface_id %} style="display:none"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)